.github/workflows/stories-summary.yml

```diff
@@ -14,6 +14,8 @@ jobs:
     steps:
       - name: 'Checkout code'
         uses: actions/checkout@v2
+        with:
+          ssh-key: ${{ secrets.DEPLOY_KEY }}
```

Comment:
Done.

Would you mind if we merged or renamed the summary files? `.github/workflows` is growing more and more and it's becoming easier to miss files. Either to `summary-stories.yml` and `summary-languages.yml` or merge it into one file with two jobs, like `valid-files.yml`

Reply:
Yeah sure. I wouldn't mind having just one file. It could even be two steps instead of separate jobs, to re-use the container. I'm not sure about the `valid-files.yml` naming. Maybe something like `update-summary-files.yml` or something similar?

Reply:
>  I'm not sure about the valid-files.yml naming.

I meant `valid-files.yml` as an example of workflows being merged into multiple jobs, not as a name suggestion.

Reply:
Oh whoops :D